components/serverless/internal/controllers/serverless/function_reconcile.go

```diff
@@ -194,3 +205,20 @@ func (r *FunctionReconciler) readDockerConfig(ctx context.Context, instance *ser
 	}
 
 }
+
+func (r *FunctionReconciler) getRuntimeImageFromConfigMap(ctx context.Context, function *v1alpha2.Function) (string, error) {
```

Comment:
Do we need to duplicate this function: https://github.com/kyma-project/serverless/blob/18602d019c5f0a03ebaf7f12ada93ec8349f43f1/components/serverless/internal/controllers/serverless/job.go#L162-L177

Reply:
We don't, so I removed it [here](https://github.com/kyma-project/serverless/pull/929/files#diff-b48afcbd1440c4333f3d5aceffe68e0a7a6b87cdec1f302fe891f1f3f0145c89L162-L178)